### PR TITLE
fixing lollipop plot problems

### PR DESCRIPTION
**Pull request by jaybee84**
it seems I was calling plotly::renderPlotly which was interfering with the plotting for the lollipopplot's own graphics::layout function. Explicitly calling shiny::renderPlot for maftools::lollipopplot fixes the problem (at least locally).

### CODE REVIEW

**Comment by jaybee84** [comment]
Added one more gene variant visualization to this PR

File: R/mod_gene_var.R [hunk 1]
```diff
@@ -101,16 +110,46 @@ mod_gene_variant_server <- function(input, output, session, specimens){
...
-    #                    titleFontSize = 1.0, keepGeneOrder = TRUE, GeneOrderSort = TRUE, bgCol = "white", borderCol = "white")
+    
   })
   
+
+  output$onco_plot <- shiny::renderPlot({
+  
+    tumor_sample_bc <- as.vector(kairos::jhu_tumor_file@data$Tumor_Sample_Barcode[kairos::jhu_tumor_file@clinical.data$specimenID %in% specimens()])
+  
+    validate(need(length(tumor_sample_bc)>0, "No variant data found. Please modify your cohort."))
```
Comment (allaway) [inline R/mod_gene_var.R:121]: With one gene i got this error: "Oncoplot requires at-least two genes for plottng." - can you add a `validate` statement to catch this error with a 'call to action' for the user?

**Review by allaway — CHANGES_REQUESTED** [review]
I ran into a different error that can be fixed with a validate statement, but otherwise LGTM!

File: R/mod_gene_var.R (hunk 1 above)
Comment (jaybee84) [reply R/mod_gene_var.R:121]: Ah! This is specific for Oncoplot because it plots multiple genes at once. Will make that explicit.

**Review by allaway — APPROVED** [review]
LGTM!

**Pull request merged**